src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccess.tsx

```diff
@@ -0,0 +1,298 @@
+/**
+ * Copied from src/lib/containers/access_requirement_list/AcceptedRequirements.tsx
+ */
+
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import MarkdownSynapse from '../../MarkdownSynapse'
+import {
+  AccessApproval,
+  AccessRequirement,
+  ACTAccessRequirement,
+  ACTSubmissionStatus,
+  ApprovalState,
+  ManagedACTAccessRequirement,
+  SelfSignAccessRequirement,
+  SUBMISSION_STATE,
+  TermsOfUseAccessRequirement,
+  UserProfile,
+  WikiPageKey,
+} from '../../../utils/synapseTypes/'
+import { SynapseClient } from '../../../utils'
+import AccessApprovalCheckMark from '../AccessApprovalCheckMark'
+import { SUPPORTED_ACCESS_REQUIREMENTS } from '../AccessRequirementList'
+import { ManagedACTAccessRequirementStatus } from '../../../utils/synapseTypes/AccessRequirement/ManagedACTAccessRequirementStatus'
+import { cancelDataAccessRequest } from '../../../utils/SynapseClient'
+import { AlertProps } from './RequestDataAccessStep2'
+import { Alert } from 'react-bootstrap'
+
+export type RequestDataAccessProps = {
+  user: UserProfile | undefined
+  token: string | undefined
+  wikiPage: WikiPageKey | undefined
+  entityId: string
+  accessRequirement:
+    | AccessRequirement
+    | TermsOfUseAccessRequirement
+    | SelfSignAccessRequirement
+    | ManagedACTAccessRequirement
+  accessRequirementStatus?: ManagedACTAccessRequirementStatus
+  showButton?: boolean
+  onHide?: Function
+  requestDataStepCallback?: Function
+}
+
+// const LoginPage: React.FunctionComponent<LoginPageProps> =
+const RequestDataAccess: React.FC<RequestDataAccessProps> = (props) => {
+  const { user, token, wikiPage, accessRequirementStatus, accessRequirement, showButton = true, onHide, requestDataStepCallback } = props
+  const [isHide, setIsHide] = useState<boolean>(true)
+  const propsIsApproved = accessRequirementStatus?.isApproved
+  const [isApproved, setIsApproved] = useState<boolean | undefined>(
+    propsIsApproved,
+  )
+  const [submissionState, setSubmissionState] = useState<SUBMISSION_STATE>()
+  const [alert, setAlert] = useState<AlertProps | undefined>()
+  const [isSubmissionCanceled, setIsSubmissionCanceled] = useState<boolean>(false)
+
+  useEffect(() => {
+
+    setIsApproved(propsIsApproved)
+    if (accessRequirementStatus?.currentSubmissionStatus) {
+      setSubmissionState(accessRequirementStatus.currentSubmissionStatus.state)
+      showSubmissionStatusAlert(accessRequirementStatus.currentSubmissionStatus.state)
+    }
+
+  }, [propsIsApproved])
+
+  const gotoSynapseAccessRequirementPage = () => {
+    requestDataStepCallback?.({
+      managedACTAccessRequirement: accessRequirement,
+      step: 1
+    })
+  }
+
+  const onAcceptClicked = async () => {
+    if (
+      accessRequirement.concreteType ===
+        SUPPORTED_ACCESS_REQUIREMENTS.ManagedACTAccessRequirement ||
+      accessRequirement.concreteType ===
+        SUPPORTED_ACCESS_REQUIREMENTS.ACTAccessRequirement
```

Comment:
I don't think SUPPORTED_ACCESS_REQUIREMENTS.ACTAccessRequirement should be handled by any of your new code.  This type of Access Requirement does not have an access request in Synapse (it's handled out-of-band, typically either by email or Jira).

Reply:
Got it. I deleted the check for SUPPORTED_ACCESS_REQUIREMENTS.ACTAccessRequirement